### PR TITLE
fix(CX-1626, CX-1630): unable to press 'following' button

### PR DESCRIPTION
**Pull request by edez1t**
The type of this PR is: **Bugfix**

This PR resolves [CX-1626] [CX-1630]

### Description
this bug was introduced in https://github.com/artsy/eigen/pull/4851

https://user-images.githubusercontent.com/73339470/123274340-1abf4a80-d50c-11eb-9317-9274f4ee351e.mov

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

[CX-1626]: https://artsyproduct.atlassian.net/browse/CX-1626
[CX-1630]: https://artsyproduct.atlassian.net/browse/CX-1630

### CODE REVIEW

File: src/palette/elements/Touchable/Touchable.tsx [hunk 1]
```diff
@@ -16,6 +16,7 @@ interface ExtraTouchableProps {
   flex?: number
   haptic?: HapticFeedbackTypes | true
   noFeedback?: boolean
+  useReactNativeTouchable_ios?: boolean
```
Comment (MounirDhahri) [inline src/palette/elements/Touchable/Touchable.tsx:19]: **thought:** what do you think about changing this to be probably `useDefaultTouchable`

**Review by MounirDhahri — APPROVED** [review]
This looks good to me. I only left a comment about the naming

File: src/palette/elements/Touchable/Touchable.tsx (hunk 1 above)
Comment (edez1t) [reply src/palette/elements/Touchable/Touchable.tsx]: done

**Comment by edez1t** [comment]
#squashongreen

**Pull request merged**